### PR TITLE
Updating Jekyll theme to latest version (5.5.5)

### PR DESCRIPTION
**Pull request by kylekirkby**
Updating Jekyll theme to latest version (5.5.5)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
 # Jumbo Jekyll Theme 
-gem 'jumbo-jekyll-theme', "3.9.4"
+gem 'jumbo-jekyll-theme', "5.5.5"
```
Comment (morancj) [inline Gemfile:4]: PR for the `jekyll-build-container` [Dockerfile](https://github.com/linaro-its/jekyll-build-container/blob/master/Dockerfile) please. :smile: Please also update the `Used by` lines as appropriate.

File: _pages/about.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
 ---
-layout: container-breadcrumb
+layout: flow
 title: About
+description: >
+    The goal of Linaro Reference Platforms (RPs) is to provide product quality, end to end, documented, open source platforms for market segment or use cases, comprising boot firmware, kernel, distribution(s) and additional relevant open source projects
```
Comment (morancj) [inline _pages/about.md]:
```suggestion
    The goal of Linaro Reference Platforms (RPs) is to provide product quality, end to end, documented, open source platforms for market segment or use cases. The Reference Platforms comprise boot firmware, kernel, distribution(s) and additional relevant open source projects.
```

File: _pages/about.md [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
 permalink: /about/
 ---
 
 The goal of Linaro Reference Platforms (RPs) is to provide product quality, end to end, documented, open source platforms for market segment or use cases, comprising boot firmware, kernel, distribution(s) and additional relevant open source projects. RPs are built and tested on 96Boards RP-Certified hardware and (for the Enterprise RP) the Linaro Developer Cloud. The RPs are intended to be reference examples for use as a foundation for products based on open source technologies.
```
Comment (morancj) [inline _pages/about.md]:
```suggestion
 The goal of Linaro Reference Platforms (RPs) is to provide product quality, end to end, documented, open source platforms for market segment or use cases. The Reference Platforms comprise boot firmware, kernel, distribution(s) and additional relevant open source projects. RPs are built and tested on 96Boards RP-Certified hardware and (for the Enterprise RP) the Linaro Developer Cloud. The RPs are intended to be reference examples for use as a foundation for products based on open source technologies.
```

**Review by morancj — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (pcolmer) [reply Gemfile:4]: Platforms doesn't use the Docker build process yet, @morancj

File: _data/footer.yml [hunk 4]
```diff
@@ -1,35 +1,44 @@
...
+      - name: Enterprise RP
+        url : /documentation/Reference-Platform/Platforms/Enterprise/
+      - name: Enterprise RP Docs
+        url : /documentation/Reference-Platform/Platforms/Enterprise/Documentation/
+      - name: Enterprise RP Downloads
+        url : http://releases.linaro.org/reference-platform/enterprise/latest/?utm_source=platforms.linaro&utm_medium=submenu
+      - name: Enterprise RP Support
+        url : /documentation/Reference-Platform/Platforms/Enterprise/Documentation/Support/
+      - name: Consumer RP
+        url : http://localhost:4005/documentation/Reference-Platform/Platforms/Consumer/
```
Comment (pcolmer) [inline _data/footer.yml]: Errr ... not localhost!!!!

File: _data/footer.yml [hunk 5]
```diff
@@ -1,35 +1,44 @@
...
+      - name: IoT RP
+        url : http://localhost:4005/documentation/Reference-Platform/Platforms/Consumer/
```
Comment (pcolmer) [inline _data/footer.yml]: and not this one either.

**Review by pcolmer — CHANGES_REQUESTED** [review]

**Review by pcolmer — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -2,11 +2,11 @@ Welcome to the official documentation for Linaro and the Reference Software Plat
 
 ## Contents
 
-- [Reference Platform Home](Reference-Platform/README.md)
+- [Reference Platform Home](https://github.com/Linaro/documentation/blob/master/Reference-Platform/README.md)
```
Comment (pcolmer) [inline README.md:5]: That isn't right.

We should be converting README.md into index.html and then just referencing `Reference-Platform`.

File: README.md [hunk 7]
```diff
@@ -2,11 +2,11 @@ Welcome to the official documentation for Linaro and the Reference Software Plat
...
    - Get started here with the Reference Software Platform
-- [Definitions](Definitions/README.md)
+- [Definitions](https://github.com/Linaro/documentation/blob/master/Definitions/README.md)
```
Comment (pcolmer) [inline README.md:7]: Nope. See comment above.

File: README.md [hunk 8]
```diff
@@ -2,11 +2,11 @@ Welcome to the official documentation for Linaro and the Reference Software Plat
...
    - Linaro and 96Boards terminology
-- [Contribute to the Reference Platform](Reference-Platform/Contribute/README.md)
+- [Contribute to the Reference Platform](https://github.com/Linaro/documentation/blob/master/Reference-Platform/Contribute/README.md)
```
Comment (pcolmer) [inline README.md:9]: and this should be `Reference-Platform/Contribute`

**Review by pcolmer — CHANGES_REQUESTED** [review]

**Comment by pcolmer** [comment]
This project is now archived.

**Pull request closed without merging**